_build/stac.hbs

```diff
@@ -46,11 +46,19 @@
 {{#each Configurations}}
 	{{{copyJson this}}},
 {{/each}}
+{{#if DocumentationLinks }}    

```

Comment:
I would intend this so it's on the same level as the closing statement "{{/if}}"

Reply:
Thanks